test/integration/test_available_regions.rb

```diff
@@ -18,6 +18,6 @@ def test_available_regions_returns_array_of_symbols
   # This test might fail if we add new regions. Since this is an integration test
   # I am fine with that!
   def test_available_regions_returns_correct_number_of_regions
-    assert_equal 244, Holidays.available_regions.count
+    assert_equal 245, Holidays.available_regions.count
```

Comment:
This comment is weird but to me this change makes sense. Comment makes it sound like we should be failing here.